src/lightning/pytorch/loggers/wandb.py

```diff
@@ -405,6 +405,8 @@ def experiment(self) -> Union["Run", "RunDisabled"]:
                     self._experiment.define_metric("trainer/global_step")
                     self._experiment.define_metric("*", step_metric="trainer/global_step", step_sync=True)
 
+        assert isinstance(self._experiment, (Run, RunDisabled))
+        self._experiment._label(code="lightning_logger")
```

Comment:
Let's avoid calling the protected API here. If this is a feature, we can look into it in a separate PR.